apps/src/bounce/skins.js

```diff
@@ -125,22 +126,26 @@ exports.load = function (assetUrl, id) {
   };
   skin.basketball = {
     background: skin.assetUrl('basketball_background.png'),
-    ball: skin.assetUrl('basketball_ball.png')
+    ball: skin.assetUrl('basketball_ball.png'),
+    paddle: skin.assetUrl('hand_1.png'),
   };
   skin.soccer = {
     background: skin.assetUrl('soccer_background.png'),
     ball: skin.assetUrl('soccer_ball.png'),
-    drawTiles: false
+    drawTiles: false,
+    paddle: skin.assetUrl('soccer_paddle.png'),
   };
   skin.hockey = {
     background: skin.assetUrl('hockey_background.png'),
     ball: skin.assetUrl('hockey_ball.png'),
-    drawTiles: false
+    drawTiles: false,
+    paddle: skin.assetUrl('hockey_paddle.png'),
   };
   skin.football = {
     background: skin.assetUrl('football_background.png'),
     ball: skin.assetUrl('football_ball.png'),
-    drawTiles: false
+    drawTiles: false,
+    paddle: skin.assetUrl('soccer_paddle.png'),
```

Comment:
Aren't these "hand" "foot" and "hockey_stick"? Also, skin.football.paddle should definitely be a football thing.